### DISCUSSION TITLE
WhatsApp Business

### DISCUSSION THREAD

**Discussion by techtoch1** [chatwoot/chatwoot#8849] [Self hosted]
Hello guys,

We are looking for someone to help us connect the WhatsApp Business app to Chatwoot. We want to be able to scan a QR code instead of just using the API.

Chatwoot is hosted on our server and any help would be greatly appreciated. We are also willing to pay to have this done.

**Comment by amandeepsinghkhalsa** [comment]
@techtoch1 Hello did you solve the issue?
https://biolinkplus.com/Meta-API-Expert
